<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(196, 255, 248);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Himalaya</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="timer">
       <property name="font">
        <font>
         <family>Microsoft Himalaya</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(172, 172, 172);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>105</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Himalaya</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>231</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="timerButton">
       <property name="font">
        <font>
         <family>Sitka Display</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
background-color:rgb(0, 255, 0);
border:none;
border-left: 1px solid rgb(89, 255, 64);
border-right: 1px solid rgb(0, 161, 27);
border-bottom: 4px solid rgb(0, 161, 27);

}
QPushButton:hover{
background-color:rgb(110, 255, 115);
border-left: 1px solid rgb(89, 255, 64);
border-right: 1px solid rgb(0, 161, 27);
border-bottom: 4px solid rgb(0, 161, 27);

}

QPushButton:pressed{
background-color: rgb(0, 117, 19);
border-left: 1px solid rgb(89, 255, 64);
border-right: 1px solid rgb(0, 161, 27);
border-bottom: 4px solid rgb(0, 161, 27);

}

</string>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Sitka Display</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(255, 0, 0);
border: none;
border-left: 1px solid rgb(255, 60, 60);
border-right: 1px solid rgb(175, 8, 8);
border-bottom: 4px solid  rgb(175, 8, 8);

}


QPushButton:hover{
background-color:rgb(255, 70, 70);
border-left: 1px solid rgb(255, 60, 60);
border-right: 1px solid rgb(175, 8, 8);
border-bottom: 4px solid  rgb(175, 8, 8);

}

QPushButton:pressed{
background-color:rgb(139, 7, 7);
border-left: 1px solid rgb(255, 60, 60);
border-right: 1px solid rgb(175, 8, 8);
border-bottom: 4px solid  rgb(175, 8, 8);
}</string>
       </property>
       <property name="text">
        <string>RESET</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>162</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Microsoft Himalaya</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>DATA:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="date">
       <property name="font">
        <font>
         <family>Microsoft Himalaya</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>136</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Microsoft Himalaya</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ORA:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="displayedTime">
       <property name="font">
        <font>
         <family>Microsoft Himalaya</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">alternate-background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
